### DISCUSSION TITLE
CHIA(1.1.6) NETWORK SYNCING PROBLEM - DEBUG FILE BELOW

### DISCUSSION THREAD

**Discussion by gokhansnbl1** [Chia-Network/chia-blockchain#5692] [Support]
I have syncing problem with Chia. I removed and installed it again. Here's my debug file: 
2021-05-21T13:06:54.759 harvester chia.harvester.harvester: WARNING  Not farming any plots on this harvester. Check your configuration.
2021-05-21T13:07:00.171 wallet chia.rpc.wallet_rpc_api    : ERROR    error No backup on backup service
2021-05-21T13:07:10.349 wallet chia.rpc.wallet_rpc_api    : ERROR    error No backup on backup service
2021-05-21T13:13:00.764 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia\full_node\full_node.py", line 640, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 121.45.204.219

2021-05-21T13:18:08.292 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia\full_node\full_node.py", line 640, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 220.135.187.152

2021-05-21T13:24:43.846 full_node full_node_server        : WARNING  Banning 188.32.226.228 for 600 seconds
2021-05-21T13:24:43.847 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia\full_node\full_node.py", line 640, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 188.32.226.228

There seems to be problem with peer. Any solution?

**Comment by MaximusChia** [comment]
I have same on My Logs,
I am syncing but only from a single peer where i have more than 20 Peers connected to my system actually

**Reply by gokhansnbl1** [reply]
I deleted all files about Chia ( from AppData and C://User ), then installed it again. Now, it works well and synced in 21 hours from beginning.

**Reply by room101-dev** [reply]
> I deleted all files about Chia ( from AppData and C://User ), then installed it again. Now, it works well and synced in 21 hours from beginning.

Impossible, I started on a fresh UBUNTU 20 system, with two m.2. NVME drives last week, I started from scratch, last week it took four full days to download the block-chain, and it never did sync.

WHen you say you deleted all files, do you know about ~/.chia/main-net? I think not, because this week it would have taken five days, and I also have a full speed fiber-optic network to my ISP. Right now the chia is only 3gb in main-net, no reason in hell for it too take 4-5 days.

Deleting all files means deleting everything on that system related to chia, it means you need to find the main-net and delete that path, its obviously hidden from you on 'windows'.

The only system I have that sync's on my lan, is my original linux mint system that I setup 2+ months ago, I don't run it any more because its now my plotter, and the 'chia start -r all' daemon, cannot co-exist with the plotter, so I now run on separate machines, I also run the chia-daemon on an old pc with ubuntu-18; I works as my harvester to chia-net

Last week, the last time I tried to install chia, none of the install*.sh script files seem to be working, they all seem to be developed on macintosh, written for all possible OS, and none of them tested. They still don't have a requirements list of python so that we can actually just install this shit outselves correctly they mis-use nodejs&npm, and the there is no mention of what version to use, they obviously don't know either.
The documentation should include a seed-file which lists all mother IP"s to bootstrap the main-net on your computer, they don't list this either;
They have not update the install wiki  for 3+ months, well at least for linux; Never have I ever seen the install-gui.sh correctly work, I have to always read the .sh and manually install everything myself.

The GUI & INSTALL scripts do not work.

**Reply by gokhansnbl1** [reply]
what will happen if i lie? nothing. as i said, i deleted all the files about the chia in my pc. Then, it quickly synced. i don't know what was the reason for not syncing, somehow it worked.

**Comment by BrandtH22** [comment]
@gokhansnbl1 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
